package.json

```diff
@@ -11,8 +11,10 @@
 		"check-node-version": "^3.1.1",
 		"codecov": "^2.3.1",
 		"glob": "^7.1.2",
+		"is-plain-obj": "^1.1.0",
```

Comment:
Does this need to be defined as a `devDependencies` in the root if we're not using it in the root?

Reply:
Package are local, tests are global, so we just need pick one approach for those dependencies. I have a feeling that other packages define them in their own `package.json` file. Lerna will hoist, them so we don't need it in here.

Reply:
Okay. Seemed we should need it to be defined in _either_ root _or_ `npm-package-json-lint-config/package.json`, but not both.